tests/tabular/checks/model_evaluation/train_test_prediction_drift_test.py

```diff
@@ -124,3 +137,57 @@ def test_drift_max_drift_score_condition_pass_threshold(drifted_data_and_model):
         details='Found model prediction PSI drift score of 0.79',
         name='categorical drift score < 1 and numerical drift score < 1'
     ))
+
```

Comment:
Minor comment 2: Can add a test for probas + multiclass that display has multiple plots